.goreleaser.yaml

```diff
@@ -5,13 +5,54 @@
 before:
   hooks:
     - go mod tidy
+
 builds:
   - env:
       - CGO_ENABLED=0
     goos:
       - linux
       - darwin
       - windows
+    goarch:
+      - amd64
+      - arm64
+
+dockers:
+  - image_templates:
+      - "ghcr.io/riskident/{{ .ProjectName }}:{{ .Version }}-amd64"
+    dockerfile: Dockerfile
+    use: buildx
+    goarch: arm64
+    goos: linux
+    build_flag_templates:
+      - --platform=linux/amd64
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
```

Comment:
Well, it's how Go does time templating :)

https://pkg.go.dev/time#pkg-constants

Don't know if you're unfamiliar, but the idea behind it is that it should be easier to remember and easier to read, because you write:

1. Month: 01, _1, Jan, or January
2. Day of month: 02 or 2
3. Hour: 03 or _3 (for 12-hour clock), or 15 (for 24-hour clock)
4. Minute: 04
5. Seconds: 05
6. Year: 06 or 2006 (weird part having it so late in the list)
7. Timezone: Z07:00, Z0700, -07:00, +0700, or MST
8. -
9. Milliseconds: .99999999 (as many 9s as you want precision)

There are more formats for each element, but in general they tried to optimize so it's easier to review, as the format looks like a real date instead of a bunch of letters mashed together.

Personally it's not that difficult reading the `YYYY-mm-ddThh-MM-ss` format either, but oh well. Someone at Go thought they'd try reinvent time formats, and here we are.

Reply:
Thanks for the explanation. I'm a noob in Go templating :sweat_smile: